src/clj/puppetlabs/services/ca/certificate_authority_service.clj

```diff
@@ -27,9 +30,9 @@
              comidi/routes->handler)
          settings
          path
-         wrap-with-authorization-check
-         puppet-version)))
-   context)
+         (fn [request] (wrap-with-authorization-check request {:oid-map oid-mappings}))
```

Comment:
Hmm..it seems weird to me to be returning authorization middleware, which is the responsibility of the Authorization Service, from the CA service. It seems like that service should only return custom OID mappings. I'd be willing to be overridden on that though, as I think it would be great to DRY this up.

@nathanielksmith what are your thoughts?